examples/darksage.py

```diff
@@ -3570,8 +3570,8 @@ def RadiusHI(self, tree, h):
         DiscRadii = np.zeros((len(tree),30))
         SigmaHI = np.zeros((len(tree),30))
         for i in range(1,31):
-            DiscRadii[:,i-1] = (tree['DiscRadii_'+str(i)]+tree['DiscRadii_'+str(i-1)])/2
-            SigmaHI[:,i-1] = tree['DiscHI_'+str(i)]/(np.pi*(tree['DiscRadii_'+str(i)]**2-tree['DiscRadii_'+str(i-1)]**2))*1e-2/h
+            DiscRadii[:,i-1] = (tree['DiscRadii_'+str(i)]+tree['DiscRadii_'+str(i-1)])*0.5
+            SigmaHI[:,i-1] = tree['DiscHI_'+str(i)]/(np.pi*(tree['DiscRadii_'+str(i)]**2-tree['DiscRadii_'+str(i-1)]**2))*1e-2*h
```

Comment:
@arhstevens Remind me - what's the 1e-2 factor for?

Reply:
Internal units for mass are 10^10/h Msun, internal units for distance are 10^6/h pc.  SigmaHI here needs to be in Msun/pc^2, hence the factor of (10^10/h) / (10^6/h)^2 = h/100